codecov.yml

```diff
@@ -0,0 +1,5 @@
+---
+coverage:
+  range: 0..60
+  round: down
+  precision: 2
```

Comment:
I think we can delete the codecov file completely.

Reply:
I agree. Unnecessary here. Will update MR.